tasks.py

```diff
@@ -11,6 +11,43 @@ def build_redis(c, redis_repo_path="redis", build_args="all build_tls=yes"):
     """compile redis"""
     redispath = os.path.join(os.getcwd(), redis_repo_path, "src")
     run(f"make -C {redispath} -j `nproc` {build_args}")
+    
+@task(
+    help={
+        "osname": "operating system name (eg: Linux)",
+        "dist": "package distribution to generate (eg: bionic)",
+        "redis_repo_path": "path to pre-build redis binaries",
+        "arch": "architecture (eg: x86_64)",
+        "version": "the version string for this redis build",
+        "publish": "upload to s3, via aws s3 cp, if set",
+    }
+)
+def package_redis(c, version="", osname='macos', dist='monterey', publish=False, arch="amd64", redis_repo_path="redis"):
+    """package, a compiled redis"""
+    redispath = os.path.abspath(os.path.join(os.getcwd(), redis_repo_path, "src"))
+    dest = f"redis-{version}-{osname}-{dist}-{arch}"
+    if os.path.isdir(dest):
+        shutil.rmtree(dest)
+    os.mkdir(dest)
+    binaries = ['redis-cli', 'redis-server', 'redis-sentinel', 'redis-benchmark', 'redis-check-rdb', 'redis-check-aof']
+    
+    for b in binaries:
+        shutil.copyfile(os.path.join(redispath, b), os.path.join(dest, b))
+        
+    tarball = f"{dest}.tgz"
+    tarcmd = f"tar -czvf {tarball} {dest}"
+    run(tarcmd)
+        
+    if publish is not None:
```

Comment:
'publish=False' is set by default.
I think boolean value, even false, is not None.
is it in purpose?

Reply:
Good catch, thanks.